installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

```diff
@@ -30,3 +32,11 @@ include::modules/ipi-install-validation-checklist-for-installation.adoc[leveloff
 include::modules/ipi-install-deploying-the-cluster-via-the-openshift-installer.adoc[leveloffset=+1]
 
 include::modules/ipi-install-following-the-installation.adoc[leveloffset=+1]
+
+ifeval::[{product-version}>4.6]
```

Comment:
This doesn't work?

Reply:
@johnwilkins Per your other comment:

> The {product-version} tag does not seem to work on my local or on this PR build. Somewhere it would need a definition like :product-version: 4.8. I'm not sure what the issue is, but it is breaking the conditional text. For whatever reason, it seems to work fine in the upstream.

Have you confirmed whether this works in practice or does it need to be updated here? I do see a few other hits already merged in `master` that are using a form of `{product-version}>`. Which build did you mean by "in the upstream"?